public/index.html

```diff
@@ -243,6 +243,28 @@ <h1>Do you want early access to CoinFi News?</h1>
       }
     }
     tryHook();
+
+    $(document).ready(function() {
+				$('video.product-video').each(function(index) {
+					var video = $(this);
+					
+					video.on('error', function(e) {
+						console.log('error', e);
+					})
+
+					video.one('canplaythrough', function() {
```

Comment:
Should this be `video.on('canplaythrough', ...` ?

Reply:
That event seems to fire continuously and I only need it to fire once so it's set to one instead of on. I guess I should also set the inner event listener to one.

Reply:
Gotcha, thanks for the explanation!